src/ApiPort/ApiPort.VisualStudio/AssemblyRedirectResolver.cs

```diff
@@ -16,7 +16,7 @@ namespace ApiPortVS
     /// Because the VSIX runs inside devenv.exe, the only configuration file that is used for redirects is
     /// devenv.exe.config.We don't want to modify this machine wide configuration so we are manually setting
     /// the redirects.
-    /// More info: http://stackoverflow.com/a/31248093/4220757
+    /// More info: http://stackoverflow.com/a/31248093/4220757.
```

Comment:
I'm not a fan of these rules that make you put periods everywhere. This isn't a sentence, so it doesn't need a period. In this case, depending on the interpretation/viewer of the URL, it might include the period and end up with a broken URL. It looks like stackoverflow handles this fine, so I'll leave it to you, but I think these are low-value.

Reply:
Agreed that it is annoying and low value. Added all the periods due to azure devops build machine reporting this style warning as errors. I don't see which rule forces the ending period from https://github.com/microsoft/dotnet-apiport/blob/dev/stylecop.json. Any idea where it may come from?

Reply:
Those are driven by the new style analyzers written in Roslyn (rather than stylecop). You can configure them with .editorconfig. You should be getting the same messages in your local editor/build. There should be a way to configure them to not be noisy by right-clicking them and suppressing them. I think that this project is opting into them (I don't think you get them by default)

Reply:
Fixed by PR https://github.com/microsoft/dotnet-apiport/pull/811